### PR TITLE
WAZO-1566: add send DTMF action to calls

### PR DESCRIPTION
**Pull request by bloom1**
Depends-On: https://github.com/wazo-platform/wazo-calld-client/pull/24
Depends-On: https://github.com/wazo-platform/wazo-auth-keys/pull/22
Depends-On: https://github.com/wazo-platform/xivo-bus/pull/20

Jira-Issue: https://wazo-dev.atlassian.net/browse/WAZO-1566

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,12 @@
 # Changelog
 
+## 20.06
+
+* New API to send DTMF digits to a call
+
+  * `PUT /calls/{call_id}/dtmf`
+  * `PUT /users/me/calls/{call_id}/dtmf`
```
Comment (sduthil) [inline CHANGELOG.md:8]: Have you thought how to implement DTMF in the other direction, if we ever need it? For example to send DTMF to a trunk call?

**Comment by bloom1** [comment]
recheck

**Comment by bloom1** [comment]
recheck

**Comment by bloom1** [comment]
recheck

**Pull request merged**